### PR TITLE
Add clear mention of Phabricator scripts to README

### PR DESCRIPTION
**Pull request by ermik**
**Description of the change**
The standard user documentation, if not a terrific read (it is, if you devote some time, like any good book), is a source of extensive knowledge on configuring and managing Phabricator. For a beginner, that succeeds in setting up a Dockerized version of Phabricator (for example to try out the product), it is reasonably hard to follow the docs, that are written from a perspective of a "downloaded directory". While crossing this chasm this can be an entire Wiki (hopefully in the future), one of the most pressing issues in the first-time setup is access to utility scripts Phabricator provides. The README previously had no clear mention of where they are / how to find them even though they are critical to a successful deployment — in fact some values in the config cannot be accessed through the UI *by design.*
- added new section briefly mentioning the location of Phabricator home folder and a link to Phabricator configuration docs

**Benefits**
N/A

**Possible drawbacks**
N/A

**Applicable issues**
#84 

**Additional information**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,6 +161,8 @@ services:
     --volume /path/to/phabricator-persistence:/bitnami \
     bitnami/phabricator:latest
   ```
+## Accessing [Phabricator scripts](https://secure.phabricator.com/book/phabricator/)
```
Comment (juan131) [inline README.md]: * What do you think about: "Using Phabricator scripts"? You can add a example of use so the user understand where to find them and how to use them.
* Instead of adding the link to Phabricator User Documentation on the header, add it at the bottom of the section with a message like "Find more information on [Phabricator Official Documentation](https://secure.phabricator.com/book/phabricator/)"

File: README.md [hunk 2]
```diff
@@ -161,6 +161,8 @@ services:
...
+Standard Phabricator distribution includes a number of utility scripts to configure tools and services included with it. You can find the `/bin/` directory that Phabricator docs mention at `/opt/bitnami/phabricator/bin/` inside the container.
```
Comment (juan131) [inline README.md]: Make it more simple:

Phabricator includes some scripts intended to configure tools and services included with it. You can find them at `/opt/bitnami/phabricator/bin/` on this Docker image.

For instance, you can use the command below to configure "EXAMPLE":

```
<<SCRIPT-EXAMPLE>> ...
```

Comment (ermik) [reply README.md]: Will get to this sometime this week!

Comment (superloopnetwork) [reply README.md:165]: > Will get to this sometime this week!
Hi Ermik,

Did you have some time to display the script examples that worked for you? I'm running into the same issue that you had from before - that is, I set the environment in the docker-compose file and run it but it never registers...

Comment (ermik) [reply README.md:165]: I have successfully used Bitnami image as a starting point for deploying Phabricator. Please feel free to reach out via email and we can take a look at your particular issue. ;)

Comment (superloopnetwork) [reply README.md:165]: Thank you!
